src/controlbits.rs

```diff
@@ -288,7 +288,7 @@ fn cbrecursion(
 pub(crate) fn controlbitsfrompermutation(out: &mut [u8], pi: &[i16], w: usize, n: usize) {
     assert_eq!(n, 1 << w);
     assert_eq!(pi.len(), n);
-    assert_eq!(out.len(), (((2 * w - 1) * n / 2) + 7) / 8);
+    assert_eq!(out.len(), ((2 * w - 1) * n / 2).div_ceil(8));
```

Comment:
why was ` + 7)` omitted in these cases ? shouldn't the ceil division function simply replace the `/` operator ?

I would assume `((2 * w - 1) * n / 2) + 7).div_ceil(8)` to be correct. (Applies to the subsequent changes in this PR aswell)

Reply:
ok that makes sense to me now :) thx for the elaboration